bin/git-forgit

```diff
@@ -261,7 +264,13 @@ _forgit_cherry_pick_from_branch() {
     # picked has been selected from within a branch
     while true
     do
-        branch="$(eval "$cmd" | FZF_DEFAULT_OPTS="$opts" fzf | awk '{print $1}')"
+        if [[ -z $input_branch ]]; then
+            branch="$(eval "$cmd" | FZF_DEFAULT_OPTS="$opts" fzf | awk '{print $1}')"
+        else
+            branch=$input_branch
+        fi
+
+        unset input_branch
         [[ -z "$branch" ]] && return 1
 
         _forgit_cherry_pick "$branch"
```

Comment:
We should somehow bypass the while loop here when there is an argument given. Otherwise quitting the fzf dialog via `ctrl+c` shows the branch selection (which hasn't been shown before), while I would expect it to return to the command line directly.

I would suggest adding another condition to the last line of the loop which will return the exitval if the branch was given via argument.